frame/support/test/tests/benchmark_ui/bad_param_range.stderr

```diff
@@ -0,0 +1,5 @@
+error: The start of a `ParamRange` must be less than or equal to the end
+  --> tests/benchmark_ui/bad_param_range.rs:10:11
+   |
+10 |     fn bench(x: Linear<3, 1>) {
+   |              ^
```

Comment:
Could you make it so that it points to `Linear<3, 1>` or even just whatever's in the angle brackets?

Reply:
got it like this now and fixed a few others :)

```
error: The start of a `ParamRange` must be less than or equal to the end
  --> tests/benchmark_ui/bad_param_range.rs:10:21
   |
10 |     fn bench(x: Linear<3, 1>) {
   |                        ^
┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈
```